backend/src/main/java/planit/people/preparation/DTOs/DTO_NewEventDetail.java

```diff
@@ -11,7 +11,8 @@ public record DTO_NewEventDetail(
         String description,
         String calendar_name,
         List<String> attendees,
-        @JsonFormat(pattern="yyyy-MM-dd HH:mm:ss") Date startDate,
-        @JsonFormat(pattern="yyyy-MM-dd HH:mm:ss")Date endDate,
+        String owner,
```

Comment:
How is owner represented as a string? Maybe rename it to `ownerEmail`

Reply:
owner_email